docs/source/overview.md

```diff
@@ -0,0 +1,81 @@
+# SimPhoNy: interoperability powered by ontology
+
+SimPhoNy is an ontology-based framework aimed at enabling interoperability between different simulation and data management tools, with a focus on materials science.
+
+
+
+## What can SimPhoNy be used for?
+
+### Manipulate ontology-based linked data, a format well suited for [FAIR data principles](https://en.wikipedia.org/wiki/FAIR_data)
+
+
+[Linked data](https://en.wikipedia.org/wiki/Linked_data) is a format for structured data that facilitates the interoperability among different data sources. In particular, the data is structured as a directed graph, consistent of nodes and labeled arcs. With SimPhoNy, you can not only manipulate this linked data, **but also transform existing non-linked data into linked data**.
+
+To better understand the idea of linked data, take a quick glance at the toy example below. It shows data about a city from three different data sources: the city's traffic authority, a map from a city guide, and the university registry. As some of the concepts are present in multiple datasets, the linked data representation naturally joins all of them into a single one. 
+
+<figure style="display: table; text-align:center; margin-left: auto; margin-right:auto">
+
+![Sample linked data](./_static/img/mergedlinkeddata.svg)
+
+<figcaption style="display: table-caption; caption-side: bottom; text-align:center">
+
+_Linked data about a city from three different sources: the city's traffic authority, a map from a city guide, and the university registry. Each data source is represented using a different color and column._
+
+</figcaption>
+
+</figure>
+
+
+Moreover, in SimPhoNy, all linked data is based on [ontologies](https://en.wikipedia.org/wiki/Ontology_(information_science)), which give **meaning** to the data. Specifically, SimPhoNy works with ontologies based on the [Web Ontology Language](https://en.wikipedia.org/wiki/Web_Ontology_Language), making the data compatible with the [Semantic Web](https://en.wikipedia.org/wiki/Semantic_Web).
+
+### Fetch data from a database, run a simulation and immediately store the results
+
+Ontology-based linked data is not only well suited for the interoperability of data, but also of software tools. In SimPhoNy, one can instantiate individuals from special ontology classes called _wrappers_. These wrappers are both an ontology individual, and a software interface between the core of SimPhoNy (ontology based) and external software tools. We have already developed a few wrappers for some software tools, but you can even [develop your own](./wrapper_development.md)!
```

Comment:
> ontology classes called _wrappers_

Referring to wrappers as classes seems to me quite partial. Perhaps using the term "(plug-in) mechanism" is a better approach that captures what it's more about.

Reply:
Well I do not think that calling them simply ontology classes is wrong. The idea that I want to explain here is that the user will interact with the wrappers as ontology classes, even if they are more than that. That's why I added the next sentence. Maybe we can change this next sentence by the following one (or something along the same lines).

_These wrappers are in fact a software interface between the core of SimPhoNy (ontology based) and external software tools, disguised to the user as an ontology class._